images/ai-ide/README.md

```
@@ -0,0 +1,112 @@
+## AI Theia Extensions
+
+The following npm packages are added on top of the Base-IDE dependencies:
+
+| Package | Version | Description |
+|---------|---------|-------------|
+| `@theia/ai-core` | 1.70.0 | Theia AI core framework — provides the agent/LLM infrastructure |
+| `@theia/ai-openai` | 1.70.0 | OpenAI provider integration for Theia AI |
+| `@fangoling/ai-terminal-assistant` | 0.1.0 | EduIDE Terminal Assistant — AI-powered terminal help and command suggestions |
+
+## Plugins shipped with the AI-IDE
+The Basis IDE includes a limited set of plugins to maintain its lightweight nature. As recommended by the creators of Theia, the plugins are sourced from their `builtin-extension-pack` available [here](https://open-vsx.org/extension/eclipse-theia/builtin-extension-pack). 
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)